Makefile

```diff
@@ -49,35 +44,35 @@ $(PROTOC):
 	rm $(PROTOC).zip
 	chmod 755 $(PROTOC)
 
-$(DEP):
-	mkdir -p $(TARGET)
-	$(CURL) -Lso $(DEP) $(DEP_URL)
-	chmod 755 $(DEP)
-
 Cargo.lock: Cargo.toml rs/Cargo.toml
 	$(CARGO) fetch
 
 .PHONY: rs
 rs: Cargo.lock
 	$(CARGO_TEST)
 
-.PHONY: godep
-godep: Gopkg.toml $(DEP)
-	$(DEP) ensure
+.PHONY: go-build
+go-build:
+	$(GO) build ./...
 
 .PHONY: go
-go: godep $(PROTOC)
-	$(GO) install ./vendor/github.com/golang/protobuf/protoc-gen-go
-	rm -rf go/*
+go: go-build $(PROTOC)
+	$(GO) install github.com/golang/protobuf/protoc-gen-go
 	$(PROTOC_GO) proto/destination.proto
 	$(PROTOC_GO) proto/http_types.proto
 	$(PROTOC_GO) proto/identity.proto
 	$(PROTOC_GO) proto/net.proto
 	$(PROTOC_GO) proto/tap.proto
 
+.PHONY: move-proto
+move-proto:
+	rm -rf go/*
+	mv $(MODULE_NAME)/go/ .
+	rm -rf github.com/
```

Comment:
Does this need to be distinct from the `go` directive? It seems we always want to do this so that `make go` leaves the repo in a workable state.

Reply:
Nope good point. I have moved this into `go` directive.